code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1147,6 +1147,12 @@ Traitors and the like can also be revived with the previous role mostly intact.
 			if(!puzzle_imprison(target))
 				to_chat(usr,"<span class='warning'>Imprisonment failed!</span>", confidential=TRUE)
 				return
+		if(ADMIN_PUNISHMENT_PIE)
+			var/mob/living/carbon/C = target
+			var/confirm = alert(usr, "Send honk message?", "Honk Message", "Yes", "No")
+			if(confirm == "Yes")
```

Comment:
```suggestion
			var/confirm = alert(usr, "Send honk message?", "Honk Message", "Yes", "No")
			if(confirm == "Yes")
			var/mob/living/carbon/C = target
```
Why:
If you quit the game before answering your text box that target variable will always be set.
Does it matter in the long run? No, but it looks prettier